fala/apps/adviser/tests/test_search_page_journeys_playwright.py

```diff
@@ -22,9 +14,25 @@ def test_postcode_search_journey(self):
         ]
         for postcode in test_cases:
             with self.subTest(postcode=postcode):
-                self.visit_search_page()
-                self.page.get_by_label("Postcode").fill(postcode)
-                self.page.get_by_role("button", name="Search").click()
-                expect(self.page.locator("h1")).to_have_text("Search results")
-                expect(self.page.get_by_role("listitem").filter(has_text=f"Postcode: {postcode}")).to_be_visible()
-                expect(self.page.get_by_text("in order of closeness")).to_be_visible()
+                page = self.visit_results_page(postcode)
+                expect(page.h1).to_have_text("Search results")
+                expect(page.listitem_for(postcode)).to_be_visible()
+                expect(page.item_from_text("in order of closeness")).to_be_visible()
+
+    def test_invalid_postcode_journey(self):
+        page = self.invalid_postcode_search()
+        expect(page.error_summary).to_be_visible()
+
+    def test_full_search_journey(self):
+        page = self.visit_results_page_with_full_search(
+            "SE11", "Islington Law Centre", ["Housing Loss Prevention Advice Service"]
+        )
+        expect(page.h1).to_have_text("Search results")
+        # this selector matches multiple things so picking out the 2nd one in the list.
+        expect(page.change_search_grey_box.nth(1)).to_have_text("Organisation: Islington Law Centre")
+
+    def test_no_results_journey(self):
+        page = self.visit_search_page()
+        page.organisation_input_field.fill("Test")
+        page.search_button.click()
+        expect(page.no_results_alert).to_be_visible()
```

Comment:
you could test for the content of the alert too if you want to make it extra regression proof